conformance-tests/src/main.rs

```diff
@@ -1,316 +1,50 @@
-use std::collections::HashMap;
+mod runtime;
 
-use anyhow::anyhow;
-use anyhow::Context as _;
-use bindings::{exports::wasi::http::types::HeaderError, VirtualizedApp};
-use test_environment::http::Response;
+use runtime::SpinTest;
 
-mod bindings {
-    wasmtime::component::bindgen!({
-        world: "virtualized-app",
-        path: "../host-wit",
-        with: {
-            "wasi:io": wasmtime_wasi::bindings::io,
-            "wasi:clocks": wasmtime_wasi::bindings::clocks,
-        }
-    });
-}
+// We don't use port 80 because URL parsing treats port 80 special
+// and will not include it in the URL string representation which breaks the test
```

Comment:
Is this fixed by https://github.com/bytecodealliance/wasmtime/pull/8563 ?

Reply:
I think we'll still run into because we're parsing the URL in the test component.